src/http_server.rs

```diff
@@ -1369,11 +1365,15 @@ mod tests {
         sqlite::init_default_users().await;
         // create router
         let router = create_router();
+
         // root without auth
-        let client = TestServer::new(router.await).expect("new TestServer");
+        let mut client = TestServer::new(router.await).expect("new TestServer");
+        client.do_save_cookies();
+        client.expect_success();
+
         let res = client.get("/").await;
-        assert_eq!(res.status_code(), StatusCode::OK);
```

Comment:
These `status_code() == StatusCode::OK` checks can be skipped by having the server `client.expect_success()`, which adds them for you to every request.